src/lib/signerVerifier.js

```diff
@@ -71,6 +85,35 @@ const signer = async (options) => {
   return signerImpl;
 };
 
+/**
+ * Creates a verifier based on the information provided
+ * @param did The issuer DID
+ * @param verificationMethod The verification method used to lookup the key
+ */
+const verifier = async (did, verificationMethod) => {
+  const canSignFor = await didUtil.canSign(did, verificationMethod);
+  if (!canSignFor) {
+    // always return false
+    return {
+      verify: () => false,
+    };
+  }
+
+  const [vmDid] = verificationMethod.split('#');
+  const document = await didUtil.resolve(vmDid);
```

Comment:
Should the verifier resolve the did or the `verify` method? I'd be a little bit afraid that people operate on old DID states when they keep the verifier around for more then one verification

Reply:
This method is called every time the user calls to verify the signature (in `src/creds/VerifiableCredential.js` above)